chainer/gradient_check.py

```diff
@@ -33,8 +33,8 @@ def numerical_grad(
     ``eps``.
 
     Args:
-        f (function): Python function with no arguments that runs forward
-            computation and returns the result.
+        f (`Python function`_): Python function with no arguments\
```

Comment:
How about "`callable`" rather than "Python function"? It's more precise.
Same for other instances.